packages/ui/src/design-system/Menu/BadgesMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import {Badge} from '@design-system/Badge';
+import classNames from 'classnames';
+import {ButtonMenu} from './ButtonMenu';
+import {NotificationButton} from '@design-system/Button';
+
+export type BadgeType = {
+  /** Label du badge */
+  label: string;
+  /** Détecte la suppression du badge */
+  onClose?: () => void;
+};
+
+type BadgesMenuProps = {
+  /** Comportement par défaut avec menu flottant sous le bouton, ou comportement custom activé par le click */
+  variant?: 'default' | 'custom';
+  /** Permet la customisation de l'icône */
+  icon?: string;
+  /** Liste des badges à afficher */
+  badgesList: BadgeType[];
+  /** Contenu du menu (uniquement avec le variant `default`) */
+  menuContent?: React.ReactNode;
+  /** Surcouche sur la classname du menu */
+  className?: string;
+  /** Détecte le click sur le bouton */
+  onClick?: () => void;
+  /** Détecte le clear sur l'ensemble des badges */
+  onClearBadges?: () => void;
+};
+
+export const BadgesMenu = ({
+  variant = 'default',
+  icon = 'equalizer-fill',
+  badgesList,
+  menuContent,
+  className,
+  onClick,
+  onClearBadges,
+}: BadgesMenuProps) => {
+  return (
+    <div className={classNames('flex justify-between items-start', className)}>
+      <div className="flex gap-2 flex-wrap">
+        {/* Liste des badges */}
+        {!!badgesList &&
+          badgesList.map(badge => (
+            <Badge
+              key={badge.label}
+              title={badge.label}
+              size="sm"
+              state="standard"
+              uppercase={false}
+              light
+              onClose={badge.onClose}
+            />
+          ))}
+
+        {/* Badge pour effacer tous les filtres sélectionnés */}
+        {!!badgesList && badgesList.length > 0 && !!onClearBadges && (
+          <div
+            onClick={onClearBadges} // tout le badge est clickable
+            className="cursor-pointer"
+          >
+            <Badge
+              title="Supprimer tous les filtres"
+              icon="delete-bin-6-line"
+              iconPosition="left"
+              size="sm"
+              state="grey"
+              uppercase={false}
+              light
+              onClose={() => {}}
+            />
+          </div>
+        )}
+      </div>
+
+      {/* Bouton par défaut avec ouverture du menu flottant */}
+      {variant === 'default' && (
```

Comment:
Je ne comprends pas trop la plus-value d'avoir deux variants `default` et `custom` alors que le composant `ButtonMenu` utilise déjà un `NotificationButton`.
Pour l'un il faut définir les `menuContent` et pour l'autre le `onClick`. Je suis pas sûre que l'on est besoin de faire ces variants dans ce composant.
De plus, si l'on veut un bouton de trigger avec du texte, il faudrait rajouter une props, idem si l'on veut mettre le bouton en `disabled` etc.
Si l'on met une props `triggerButton: React.ReactNode` afin de donner le composant que l'on veut et de pouvoir le custom à volonté. Qu'en penses tu?

Reply:
Je l'ai fait de cette manière pour être sûre d'avoir une homogénéité dans la mise en forme. Si le composant button est fourni en props, ce n'est pas garanti. On pourrait passer des props par défaut pour le bouton depuis ce composant, mais là encore on pourrait les changer. Je trouve ça un peu dommage du coup.

Reply:
Pourquoi ce ne serait pas garanti?
Il y a un className sur le container global qui permet d'ajuster les styles.

De plus, actuellement le bouton par défaut nécessite un `menuContent` or il n'est pas obligatoire dans les props.